ivy/functional/frontends/numpy/__init__.py

```diff
@@ -490,6 +489,8 @@ def promote_types_of_numpy_inputs(
 from .matrix import *
 from . import random
 from .random import *
+from . import indexing_routines
```

Comment:
This import should be left where it was as this is an unnecessary change right?🙂

Reply:
Hi @hirwa-nshuti, this was to avoid circular imports